pkg/router/buyer.go

```diff
@@ -13,8 +15,11 @@ import (
 // @Success 200
 // @Failure 401
 // @Router /buyer/order [get]
-func buyerGetOrderHistory(c echo.Context) error {
-	return c.NoContent(http.StatusOK)
+func buyerGetOrderHistory(c *db.DB, logger *zap.SugaredLogger) echo.HandlerFunc {
+	return func(c echo.Context) error {
+
+		return c.NoContent(http.StatusOK)
+	}
```

Comment:
should unified the parameter name to db or another

Reply:
fixed